lib/SILGen/SILGenBridging.cpp

```diff
@@ -2225,7 +2225,8 @@ void SILGenFunction::emitForeignToNativeThunk(SILDeclRef thunk) {
                                                 foreignFormalType,
                                                 foreignLoweredTy);
         // Handle C pointer arguments imported as indirect `self` arguments.
```

Comment:
This comment is no longer correct. Or maybe this isn't the right place for this logic?

Reply:
Agreed. I am not even sure what the comment is trying to clarify here.